File: java/src/automation-root/automation-testing/src/test/java/com/automation/testing/cases/CreateStudent.java
package com.automation.testing.cases;

import com.automation.api.pages.Students;
import com.automation.core.testing.TestCase;

import java.io.IOException;
import java.lang.reflect.InvocationTargetException;
import java.time.LocalDateTime;
import java.util.Map;

public class CreateStudent extends TestCase {
    @Override
    public boolean automationTest(Map<String, Object> testParams)
            throws ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException, IOException {

        // create driver for this test case
        String application = testParams.get("application").toString();
        String firstName = testParams.get("firstName").toString();
        String lastName = testParams.get("lastName").toString();
        String fluent = testParams.get("fluent").toString();
        String students = testParams.get("students").toString();

        return createFluentApi(fluent)
                .<Students>changeContext(students, application)
                .create()
                .firstName(firstName)
                .lastName(lastName)
                .enrollmentDate(LocalDateTime.now())
                .create()
                .findByName(firstName)
                .students()
                .size() > 0;
    }
}

File: csharp/src/AutomationRoot/Automation.Testing/Cases/CreateStudent.cs
using Automation.Api.Pages;
using Automation.Core.Components;
using Automation.Core.Testing;
using Automation.Framework.Ui.Pages;
using System;
using System.Collections.Generic;
using System.Linq;

namespace Automation.Testing.Cases
{
    public class CreateStudent : TestCase
    {
        public override bool AutomationTest(IDictionary<string, object> testParams)
        {
            // testing data
            var firstName = $"{testParams["firstName"]}";
            var lastName = $"{testParams["lastName"]}";
            var fluent = $"{testParams["fluent"]}";
            var students = $"{testParams["students"]}";

            // perform test case
            return CreateFluentApi(fluent)
                .ChangeContext<IStudents>(students, $"{testParams["application"]}")
                .Create()
                .FirstName(firstName)
                .LastName(lastName)
                .EnrollementDate(DateTime.Now)
                .Create()
                .FindByName(firstName)
                .Students()
                .Any();
        }
    }
}